README.md

```diff
@@ -202,31 +219,31 @@ Which is the way you would configure `nested_attributes` 90% of the time. Should
 A few options can be passed to `abyme.records`:
 * `collection:` : allows you to pass a collection of your choice to only display specific objects.
 ```ruby
-  <%= abymize(:tasks, f) do |abyme| %>
+  <%= f.abyme_for(:tasks) do |abyme| %>
     <%= abyme.records(collection: @project.tasks.where(done: false)) %>
     <%= abyme.new_records %>
     <%= add_association %>
   <% end %>
 ```
 * `order:` : allows you to pass an ActiveRecord `order` method to sort your instances the way you want.
 ```ruby
-  <%= abymize(:tasks, f) do |abyme| %>
+  <%= f.abyme_for(:tasks) do |abyme| %>
     <%= abyme.records(order: { created_at: :asc }) %>
     <%= abyme.new_records %>
     <%= add_association %>
   <% end %>
 ```
-* `partial:` : allows you to indicate a custom partial, if one has not already been passed to `abymize`.
+* `partial:` : allows you to indicate a custom partial, if one has not already been passed to `abyme_for`.
```

Comment:
can you also write this then: 
```
<%= f.abyme_for(:tasks, partial: 'projects/task_fields') do |abyme| %>
  <%= abyme.records(order: { created_at: :asc }) %>
  <%= abyme.new_records %>
  <%= add_association %>
<% end %>
```
?

Reply:
Roger, it's documented below for the `#abyme_for` part